pages/src/ui-playground/ui-playground.less

```diff
@@ -0,0 +1,42 @@
+@import (inline) '../../node_modules/@exadel/ui-playground/esm/registration.css';
+@import (reference) '../common/variables.less';
+
+// Default theme definition
+html {
+  --uip-bg: #fff;
+  --uip-fg: @body-fg;
+
+  --uip-header-bg: #112a47;
+  --uip-header-fg: #fff;
+
+  --uip-prview-max-height: 500px;
+
+  --uip-plugin-fg: #ccc;
+  --uip-plugin-bg: @primary-bg;
+  --uip-plugin-divider: #c8d6f0;
+
+  --uip-plugin-header-bg: #112a47;
+  --uip-plugin-header-fg: #fff;
+
+  --uip-plugin-fg-v: @body-fg;
+  --uip-plugin-bg-v: transparent;
+  --uip-plugin-divider-v: #c8d6f0;
+}
+
+.uip-root {
+  height: auto;
+  margin-block-end: 1rem;
+
+  overflow: hidden;
+  border: 1px solid #c8d6f0;
```

Comment:
as a suggestion
```suggestion
  border: 3px solid @body-fg;
```

![image](https://github.com/exadel-inc/esl/assets/52083788/2fd24095-c379-42d7-8813-6bde0d67ac9d)

Reply:
Regarding borders - tricky question.
To be honest, the border is confusing me on the screen, but the original does not satisfy me either... 

I'll add that moment to the list for the followup UIP style discussion.